samtranslator/policy_templates_data/policy_templates.json

```diff
@@ -1920,6 +1920,54 @@
         }
       }
     },
+    "SESBulkTemplatedCrudPolicy_v2": {
+      "Definition": {
+        "Statement": [
+          {
+            "Action": [
+              "ses:GetIdentityVerificationAttributes",
+              "ses:SendEmail",
+              "ses:SendRawEmail",
+              "ses:SendTemplatedEmail",
+              "ses:SendBulkTemplatedEmail",
+              "ses:VerifyEmailIdentity"
+            ],
+            "Effect": "Allow",
+            "Resource": [
+              {
+                "Fn::Sub": [
+                  "arn:${AWS::Partition}:ses:${AWS::Region}:${AWS::AccountId}:identity/${identityName}",
```

Comment:
Have we tested this? Many of these actions do not support a resource constraint, which might cause them to fail if it is present. You may need to divide the actions amongst resource-constrained and `*`-constrained depending on what resource types are supported in this table: https://docs.aws.amazon.com/service-authorization/latest/reference/list_amazonses.html

Reply:
Great catch, you were right. The actions `ses:GetIdentityVerificationAttributes` and `ses:VerifyEmailIdentity` don't support resource constraints so those action will fail. They're also part of the `SESBulkTemplatedCrudPolicy` and the `SESCrudPolicy` which both have resource constrains so they'll fail with those policies as well. 

Going to take those actions out of this policy and create a separate, *-constrained, policy for them.